app/models/night_finder.rb

```diff
@@ -0,0 +1,30 @@
+class NightFinder
+  def find_nights(night_params)
+    @_night_params = night_params
+
+    Night.all_for_couch(couch_id)
+      .between_check_in_check_out(check_in, check_out)
```

Comment:
@Sh1pley @njgheorghita great call. Once we get Brett's PR in for reservation_id on nights, we can an available nights scope.